lxde-base/menu-cache/menu-cache-1.1.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Library to create and utilize caches to speed up freedesktop application menus"
+HOMEPAGE="http://lxde.sourceforge.net/"
+SRC_URI="mirror://sourceforge/lxde/${P}.tar.xz"
+
+LICENSE="GPL-2"
```

Comment:
LICENSE="LGPL-2.1+" as per https://github.com/lxde/menu-cache/blob/master/COPYING They decided to switch it on us.

Reply:
Done.